homeassistant/components/motion_blinds/cover.py

```diff
@@ -191,12 +190,14 @@ def __init__(self, coordinator, blind, device_class, sw_version):
 
         self._blind = blind
         self._api_lock = coordinator.api_lock
-        self._requesting_position = False
+        self._requesting_position: CALLBACK_TYPE | None = None
         self._previous_positions = []
+        self._update_interval_moving = 5
```

Comment:
I think you should assign in the if/else rather than assign and reassign only in the if.